pkg/logs/auditor/null_auditor.go

```diff
@@ -0,0 +1,55 @@
+package auditor
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/logs/message"
+)
+
+// NullAuditor is an auditor not doing anything with the message it received.
+// It has been introduced for the Serverless Agent which doesn't need
+// to maintain a registry.
+type NullAuditor struct {
+	channel     chan *message.Message
+	stopChannel chan struct{}
+}
```

Comment:
Interesting, I'll look into this

Reply:
That's indeed a solution, however, the original implementation of the Auditor is trying to do stuff with the registry on disk (for instance trying to read/recovery it at startup) which we don't want to do in this environment. Because of this, this `NullAuditor` is slightly more broader than just doing no-op on every message.
I think I'll keep the `NullAuditor` because of that and indeed, we may consider using it for some other tailers implementation if that makes sense.

Reply:
I agree having no operation on-disk is much more relevant here.